src/libraries/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.Http3.cs

```diff
@@ -1349,7 +1349,9 @@ public async Task DuplexStreaming_RequestCTCancellation_DoesNotApply()
             new TheoryData<string>
             {
                 { "https://cloudflare-quic.com/" }, // Cloudflare with content
-                { "https://pgjones.dev/" }, // aioquic with content
+                // This endpoint is consistently failing.
+                // ISSUE: https://github.com/dotnet/runtime/issues/63009
```

Comment:
To help string-based search:

```suggestion
                // [ActiveIssue("https://github.com/dotnet/runtime/issues/63009")]
```